taskmanager/features/task.feature

```diff
@@ -9,3 +9,32 @@
       When I create a task named "Add switch language button"
       Then there should be "1" tasks with "TODO" status
 
+    Scenario: Change task status from TODO to IN PROGRESS
+      Given there is a task named "Add switch language button" with "TODO" status
+      When I move this task to "IN PROGRESS" column
+      Then there still will be "1" task
+      And it should have status "IN PROGRESS"
```

Comment:
`And task should be assigned to "me"`

Reply:
Or 
```gherkin
Then there is a task named "Add switch language button" with "IN PROGRESS" status
```
 And it can reuse the same method like `Given`.

Reply:
Added